### ISSUE TITLE
Cannot use SimpleScreenRecorder at 1920X1080 resolution

### ISSUE THREAD

**Issue by venik212** [MaartenBaert/ssr#789]
I tried to use SimpleScreenRecorder on a Linux/Mint/Xfce (19.3) connected to a Samsung HD Smart TV (60 inch).  It works fine at high resolution, but at my default 1920X1080 pixels I cannot see or click the CONTINUE button at the bottom of the first screen.  I don't really want to run my computer at 3200X1800 just to click that button, and I cannot use the program otherwise.  Could you duplicate (or relocate) the CONTINUE button to another part of the first dialog of SSR, such as near the top?
In other ways I love the program-- thanks!
Ehud.Kaplan@gmail.com

**Comment by MaartenBaert** [comment]
The SSR window has a height of about 630 ppx, if it doesn't fit on your screen then you must be using a very high scale factor. Can you post a screenshot?

**Comment by venik212** [comment]
1) Scaling factor is 1.0 (no scaling);  I was able to fit the screen in by reducing the number of dpi (font settings) from 190 to 150.  Of course, now my fonts are smaller, but I can live with that size.  I am not sure why the SSR window cannot be resized properly, but now I can use it, so I am happy.
2) Is it possible to add a scheduler to SSR,, so it could be started and stopped when desired?  That will make it into a nice PVR.  Alternatively, if this is difficult, could you tell me how to start it in running in RECORD mode from the terminal, so I could put that in a script and use cron to time it?
   Thanks.

**Comment by MaartenBaert** [comment]
The scheduler already exists in the latest version, you must be using an outdated version. That may also explain the screen size issue, since SSR is supposed to add scrollbars in this case (though it may be confused by the fact that you have two monitors with a different size).

**Comment by venik212** [comment]
Thank you, Maarten,  This kind of quick, helpful support is the reason I use linux... ;-)
I'll try the PPA.

**Comment by venik212** [comment]
Everything works now-- which is wonderful.  Will it record if the machine is asleep (laptop lid is closed)?

**Comment by MaartenBaert** [comment]
It depends. If your laptop is configured to simply disable the monitor when the lid is closed, SSR will continue to record. If your laptop is connected to an external monitor and this monitor still works with the lid closed, then SSR should be able to record it too.

However if closing the lid activates a screensaver, SSR will probably record the screensaver instead. And of course if your laptop is configured to suspend when the lid is closed, SSR won't be able to do anything at all.

**Comment by venik212** [comment]
I am still confused about the use of the scheduling feature.  I set up a recording, but it did not happen.  The current local time is shown correctly in the scheduling window.  If I START RECORDING (using either the button or the hot key) it starts recording _immediately_(which is NOT what I want), but if I wait for the scheduler to do it, nothing happens.  
   Please help!.

**Comment by MaartenBaert** [comment]
Did you press the 'activate schedule' button?

**Comment by venik212** [comment]
That step was not mentioned anywhere-- perhaps it should appear in the schedule editing window.  Now I think it will work, although the red button appears black on my screen.

**Comment by MaartenBaert** [comment]
I agree that it's not very intuitive that editing the schedule also deactivates it, especially given the order of the buttons. I've changed this now.

**Comment by venik212** [comment]
Thank you!
